ddtrace/contrib/internal/openai/utils.py

```diff
@@ -376,25 +412,39 @@ def _tag_streamed_response(integration, span, completions_or_messages=None):
 
 
 def _set_token_metrics(span, response, prompts, messages, kwargs):
-    """Set token span metrics on streamed chat/completion responses.
+    """Set token span metrics on streamed chat/completion and Responses responses.
     If token usage is not available in the response, compute/estimate the token counts.
     """
     estimated = False
     if response and isinstance(response, list) and _get_attr(response[0], "usage", None):
         usage = response[0].get("usage", {})
-        prompt_tokens = getattr(usage, "prompt_tokens", 0)
-        completion_tokens = getattr(usage, "completion_tokens", 0)
+        if hasattr(usage, "input_tokens"):
+            input_tokens = getattr(usage, "input_tokens", 0)
+            span.set_metric("openai.response.usage.input_tokens", input_tokens)
+        else:
+            prompt_tokens = getattr(usage, "prompt_tokens", 0)
+            span.set_metric("openai.response.usage.prompt_tokens", prompt_tokens)
+
+        if hasattr(usage, "output_tokens"):
+            output_tokens = getattr(usage, "output_tokens", 0)
+            span.set_metric("openai.response.usage.output_tokens", output_tokens)
+        else:
+            completion_tokens = getattr(usage, "completion_tokens", 0)
+            span.set_metric("openai.response.usage.completion_tokens", completion_tokens)
```

Comment:
```suggestion
        if hasattr(usage, "output_tokens") or hasattr(usage, "completion_tokens"):
            output_tokens = getattr(usage, "output_tokens", 0) or getattr(usage, "completion_tokens", 0)
```
Also do we actually expect `prompt/completion_tokens` to be present in a responses usage chunk? Or is it always input/output? If the latter, we shouldn't need to check for that anyway.